src/main/java/graphql/normalized/FieldCollectorNormalizedQueryParams.java

```diff
@@ -61,14 +72,19 @@ public Builder fragments(Map<String, FragmentDefinition> fragmentsByName) {
             return this;
         }
 
-        public Builder variables(Map<String, Object> variables) {
-            this.variables.putAll(variables);
+        public Builder coercedVariables(Map<String, Object> coercedVariableValues) {
+            this.coercedVariableValues.putAll(coercedVariableValues);
```

Comment:
Any reason why we do this instead of replacing the map entirely? The bahavior here is not the same as `normalizedVariables` but the naming scheme is the same

Reply:
this was the previous behavior: the new one is replacing the entire Map. The reason why because `normalizedVariableValues` is optional, but `coercedVariablesValues` is never null. But I agree this is not the most elegant solution.